# Request
int32 target_x
int32 target_y
---
# Result
bool is_finished
---
# Feedback
int32 current_x
int32 current_y
string stat
